images/router/haproxy/conf/haproxy-config.template

```diff
@@ -33,6 +33,12 @@
 
 global
   maxconn {{env "ROUTER_MAX_CONNECTIONS" "20000"}}
+{{- $threads := env "ROUTER_THREADS"
```

Comment:
Missing `}}`.

Reply:
Hopefully that was caught by an automated test...